main.js

```diff
@@ -0,0 +1,111 @@
+// Ex 1: Thuế và lương ròng
+const getSalery = (salery) => {
+    if(salery >= 15000000){
+        console.log('Thuế thu nhập =',salery*30/100);
+        console.log('Lương ròng = ',salery*70/100);
+    }
+    else if (salery < 15000000 && salery > 7000000) {
+        console.log('Thuế thu nhập =',salery*20/100);
+        console.log('Lương ròng =',salery*80/100);
+    }
+    else if (salery < 7000000 && salery > 0) {
+        console.log('Thuế thu nhập = ',salery*10/100);
+        console.log('Lương ròng = ',salery*90/100);
+    }
+}
+
+// Ex 2: tuổi vào lớp 10 của học sinh
+function ageClass(tuoi) {
```

Comment:
```suggestion
function checkIsHighSchoolAge(age) {
```
Không dùng tiếng việt
Tên hàm phải là động từ